typescript/cli/src/commands/options.ts

```diff
@@ -95,6 +95,7 @@ export const DEFAULT_WARP_ROUTE_DEPLOYMENT_CONFIG_PATH =
   './configs/warp-route-deployment.yaml';
 
 export const DEFAULT_CORE_DEPLOYMENT_CONFIG_PATH = './configs/core-config.yaml';
+export const DEFAULT_STRATEGY_CONFIG_PATH = `${os.homedir()}/.hyperlane/strategies/default-strategy.yaml`;
```

Comment:
I wanted to mimic what `context.registry.updateChain` is doing, since it seems to save the new chain config in `.hyperlane ` folder. `os.homedir()` also used in `resolvePath` and `overrideRegistryUriCommandOption`